### PR TITLE
Separate docker-compose example into file

### PR DESCRIPTION
**Pull request by jaceksan**
+ Complete DOC for APP_DB_USER

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  vertica:
+    image: "123456789012.dkr.ecr.eu-central-1.amazonaws.com/vertica:10.0.1-5.CentOS_8.3.2011"
+    ports:
+      - "5433:5433"
+    volumes:
+      - vertica-data:/data
+      - ./.docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d/
+    environment:
+      VMART_LOAD_DATA: "${VMART_LOAD_DATA:-n}"
+      APP_DB_USER: "tiger"
```
Comment (michalvanco) [inline docker-compose.yaml:13]: use the other user in example, there is no mention about anything related to `tiger`

File: README.md [hunk 2]
```diff
@@ -141,25 +126,20 @@ pip3 install requirements_tests.txt
...
+    ```
+    docker run -p 5433:5433 -d \
+      -e TZ='Europe/Prague' \
+      123456789012.dkr.ecr.eu-central-1.amazonaws.com/vertica:10.0.1-5.CentOS_8.3.2011
+    ```
+- Docker-compose
+    - [docker-compose.yaml example](docker-compose.yaml)
+    - Run
+    ```
+    docker-compose up -d vertica
```
Comment (michalvanco) [inline README.md:141]: in the context of this section and example you can also put something like `docker-compose up -e VMART_LOAD_DATA=y vertica`

File: README.md [hunk 3]
```diff
@@ -168,9 +148,11 @@ List of available configuration parameters:
   - More info about the schema can be found on [official web site](https://www.vertica.com/docs/10.0.x/HTML/Content/Authoring/GettingStartedGuide/IntroducingVMart/IntroducingVMart.htm)
 2. APP_DB_USER
   - Name of additional database user, who should be created
+  - This user is created only if this variable is set
```
Comment (michalvanco) [inline README.md:151]: if not set, which used should be used for connecting to vertica, `vertica` / `dbadmin`

maybe having an example how to connect to running vertica in container would answer such question

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: README.md (hunk 3 above)
Comment (jaceksan) [reply README.md:151]: Added new section into DOC describing how to connect to Vertica from outside container

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by michalvanco — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/docker-image-for-vertica-fossa-scanning-zuul-el7/8/ : SUCCESS in 10s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/docker-image-for-vertica-merge-compliance-check/8/ : SUCCESS in 17s

**Pull request merged**